### DISCUSSION TITLE
What to buy with MATLAB ? -- Global optimization toolbox OR Optimization toolbox ?

### DISCUSSION THREAD

**Discussion by SunilAnandatheertha** [mtex-toolbox/mtex#1508] [Ask Anything]
Hi, 

Please clarify whether it is the Global optimization toolbox or the Optimization toolbox which is needed to run MTEX?

I see in the below, mention of global optimization toolbox and also optimization toolbox. Please clarify which one is needed.
1. https://github.com/mtex-toolbox/mtex/discussions/1452
2. https://github.com/mtex-toolbox/mtex/discussions/604

Also, please let me know if there are any other dependencies associated so that I buy the right MATLAB product.

Kind regards
Sunil

**Comment by kilir** [comment]
Hi Sunil,
if you'd like to use calcDislocationDensities or calcTaylor, you'll need the Optimization toolbox. If you do not plan to use these, Mtex should actually also work on plain Matlab.

Cheers,
Rüdiger

**Reply by SunilAnandatheertha** [reply]
Hi,

Thank you for this reply. Yes, I will be using the GND functions.

Just wanted to make sure its not the Global Optimization toolbox that is necessary.

Kind regards
Sunil
